client/db/src/lib.rs

```diff
@@ -823,6 +823,37 @@ where
 	}
 }
 
+struct Frozen<T: Clone> {
+	at: std::time::Instant,
+	value: T,
+}
+
+pub(crate) struct FrozenForDuration<T: Clone> {
```

Comment:
This is used to always return last 1-second stats

Reply:
A comment would be nice to explain what this does :)